libs/category/src/drivers/magento/transformers/custom-metadata-attributes-transformer.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+import { MagentoAggregation } from '../models/aggregation';
+import { MagentoCustomMetadataAttribute } from '../models/requests/custom-metadata-attribute';
+import { MagentoCustomAttributeMetadataResponse } from '../models/custom-attribute-metadata-response';
+import { MagentoCompleteCategoryResponse } from '../models/complete-category-response';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoCustomMetadataAttributeTransformerService {
+
+  transform(filter: MagentoAggregation): MagentoCustomMetadataAttribute {
+		return {
+			attribute_code: filter.attribute_code,
+			entity_type: '4'
```

Comment:
Remove this, we don't use it.

Reply:
entity_type is required for the custom metadata call. Doesn't say in the Magento docs, but the call fails without it.

Reply:
It also always seems to be "4". I've tried all other values (valid values are 0-9 or 1-10, I forget), and the only one that seems not to error is "4"